### DISCUSSION TITLE
Somewhat off topic

### DISCUSSION THREAD

**Discussion by NikoV5** [SpotCompiled/SpotilifeC#21] [General]
Just curious how do you compile your IPA's to work with any sideloading tool (altstore,SideStore,sideloadly)? Whenever I inject the tweaks myself, I can only sideload using sideloadly but I would preferably like to use altstore or SideStore. Ik it's off topic but I'm genuinely confused how you managed to compile Spotify for it to work that way.

**Comment by yodaluca23** [comment]
> Just curious how do you compile your IPA's to work with any sideloading tool (altstore,SideStore,sideloadly)? Whenever I inject the tweaks myself, I can only sideload using sideloadly but I would preferably like to use altstore or SideStore. Ik it's off topic but I'm genuinely confused how you managed to compile Spotify for it to work that way.

It's supposed to work with any tool, though for some reason, it doesn't seem to work on TrollStore...

**Reply by NikoV5** [reply]
Well yours does work with any tool. But when I inject the tweak myself into later updated versions of Spotify I can only get it to work with sideloadly. I get a staging error when trying to side load on SideStore which is why I was curious what you might have did differently with yours to get it to work with everything.

**Reply by yodaluca23** [reply]
> Well yours does work with any tool. But when I inject the tweak myself into later updated versions of Spotify I can only get it to work with sideloadly. I get a staging error when trying to side load on SideStore which is why I was curious what you might have did differently with yours to get it to work with everything.

Later versions of Spotify do not work.

**Reply by NikoV5** [reply]
It does work. I got it sideloaded via sideloadly working as intended but I get error messages with altstore and staging errors with SideStore. I believe it not working on altstore and SideStore have to do with entitlements or something I was just curious if you knew how to fix that

**Reply by yodaluca23** [reply]
> It does work. I got it sideloaded via sideloadly working as intended but I get error messages with altstore and staging errors with SideStore. I believe it not working on altstore and SideStore have to do with entitlements or something I was just curious if you knew how to fix that

Spotify changed stuff in version past 8.6.42, causing Spotilife to not work, Julioverne needs to update it, so even if you could successfully compile it it would just crash, or not have any of the features...

**Reply by NikoV5** [reply]
Are you saying that's just the case when sideloading on altstore and SideStore? Because I got spotilife to work on the **latest** version of Spotify on my iPad sideloaded via sideloadly. All the features from the tweak work as intended.
